components/dispatcher/cmd/main.go

```diff
@@ -39,12 +42,16 @@ var (
 // The Main Function (Go Command)
 func main() {
 
-	// Initialize The Logger
-	logger = log.Logger()
-
 	// Parse The Flags For Local Development Usage
 	flag.Parse()
 
+	// Initialize A Knative Injection Lite Context (K8S Client & Logger)
+	ctx := commonk8s.LoggingContext(context.TODO(), Component, *masterURL, *kubeconfig)
+
```

Comment:
You used context.Background in the channel.. lets use that here as well.  TODO is supposed to be used just when you don't know which context makes sense.

Reply:
Yeah, that's because the Channel was already using Background, but I agree - will change.